environment.texi

```diff
@@ -28,7 +46,7 @@ accepts Emacs Lisp input and produces bytecode that can be run by
 Emacs. The compiler itself is written in Emacs Lisp @footnote{usually,
 the compiler itself is compiled into bytecode, which avoids overflow
 problems}, and is a comparatively compact program contained in the
-files bytecomp.el and byte-opt.el.
+files @code{bytecomp.el} and @code{byte-opt.el}.
```

Comment:
We should probably mention `cconv.el`code

Reply:
Sure. Please suggest something.